mslice/app/mainwindow.ui

```diff
@@ -31,142 +31,228 @@
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
-        <number>0</number>
+        <number>1</number>
```

Comment:
Did we mean to change the default tab to the workspace manager?

Reply:
Nope, I wish QtDesigner wouldn't do this...

Reply:
I agree - it is very annoying.